nautobot/dcim/models/device_component_templates.py

```diff
@@ -98,7 +98,9 @@ class Meta:
         unique_together = ("device_type", "name")
 
     def instantiate(self, device):
-        return ConsolePort(device=device, name=self.name, label=self.label, type=self.type)
+        return ConsolePort(
+            device=device, name=self.name, label=self.label, type=self.type, description=self.description
```

Comment:
A minor nitpick: since `description` is a base property common to all `ComponentTemplateModels`, in these function calls, I'd like to see it grouped along with the other common base properties of `name` and `label`, rather than added to the end of the call (after the model-specific properties).

```suggestion
            device=device, name=self.name, label=self.label, description=self.description, type=self.type
```